packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -190,111 +223,41 @@ export default {
       default: "",
     },
     /**
-     * Header wishlist items quantity
-     */
-    wishlistItemsQty: {
-      type: [String, Number],
-      default: "0",
-    },
-    /**
-     * Header cart items quantity
+     * Open sidebar on mobile view
      */
-    cartItemsQty: {
-      type: [String, Number],
-      default: "0",
-    },
-    /**
-     * Header sticky to top
-     */
-    isSticky: {
-      type: Boolean,
-      default: false,
-    },
-    /**
-     * Is nav slot visible on mobile view
-     */
-    isNavVisible: {
-      type: Boolean,
-      default: false,
+    openSidebar: {
+      type: [Boolean, String],
```

Comment:
Type of String? I don't think that's right ;) Let's leave just Boolean type.

Reply:
It's the thing I use in stories to open sidebar on string value. I know it's not elegant way, but I didn't find any better.

Reply:
I remove boolean type here, it's better solution as you mentioned.